public_html/wp-content/plugins/wcpt/views/applications/wordcamp/shortcode-application.php

```diff
@@ -1477,8 +1477,9 @@ class="survey-email required" autocomplete="off" required/>
 
 			</div> <!-- PDF.pageInner -->
 		</div> <!-- PDF.pageOuter -->
+		<?php wp_nonce_field( 'wcpt_event_application', 'wcpt_event_application_nonce' ); ?>
```

Comment:
- [ ] Test if nonce field is getting rendered and submitted on application submit.

Reply:
decided to not include nonce in frontend app